app/(components)/navbar/Navbar.tsx

```diff
@@ -0,0 +1,74 @@
+'use client';
+
+import Image from 'next/image';
+import Link from 'next/link';
+import { useEffect, useState } from 'react';
+import { MoonIcon, SunIcon } from '@chakra-ui/icons';
+import styles from './navbar.module.css';
+
+export default function Navbar() {
+  const [darkmode, setDarkmode] = useState(true);
+  let preferredTheme: string | null = 'dark';
+  if (typeof window !== 'undefined') {
+    preferredTheme = window.localStorage.getItem('preferred-theme');
```

Comment:
We should discuss a[ preferences object](https://discord.com/channels/1092667614863577140/1109378353737236500/1110766058253586512) instead of keeping open items so we can track and ensure preferences follow some agreed upon design. Not a blocker again, I have opened a [ task ](https://github.com/orgs/ed-pilots-network/projects/4/views/1?pane=issue&itemId=30751983)to discuss this further